components/ldap.rst

```diff
@@ -35,38 +35,95 @@ using the following options:
 ``version``
     The version of the LDAP protocol to use
 
-``useSsl``
-    Whether or not to secure the connection using SSL
+``encryption``
+    Your encryption mechanism (``ssl``, ``tls`` or ``none``)
 
-``useStartTls``
-    Whether or not to secure the connection using StartTLS
+``connection_string``
```

Comment:
Actually, the term "DSN" is specific to the database world. "connection_string" is standard for all software, and not only databases. :)